src/index.ts

```diff
@@ -63,9 +68,12 @@ function captureException<Context>(
   error: Error,
   ctx: Context,
   withScope: ExceptionScope<Context>,
+  reportError?: (res) => boolean,
 ) {
-  sentryInstance.withScope(scope => {
-    withScope(scope, error, ctx)
-    sentryInstance.captureException(error)
-  })
+  if (reportError && !reportError(err)) {
```

Comment:
I'm not completely sure here, but shouldn't the check be truthy, so if the reportError method returns true, the error should be reported? Maybe I'm missing something here though 🤔

Reply:
Good catch, I forgot to change the condition when I moved it. I also added a check if the `reportError` param isn't passed at all.